### PR TITLE
Refactoring art modules

### PR DESCRIPTION
**Pull request by sadrasabouri**
### Added
- `data` directory
- `tests` directory
### Changed
- `art_param` module changed to `params`
- `art` module split into `errors`, `utils`, and `functions` modules
- `text_dic1`, `text_dic2`, and `text_dic1` renamed to `fonts1`, `fonts2`, and `fonts3`
- `art_dic` renamed to `arts`
- `decor_dic` renamed to `decorations`
- Fonts, art, and decorations moved to `data`
- Tests moved to `tests` directory

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -395,100 +404,100 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
-	88. wave dance
-	89. whisling
-	90. winnie the pooh
-	91. winning
-	92. wizard2
-	93. worried
-	94. yun
-	95. zombie2
-	96. zoned
+  	1. angry troll
```
Comment (sepandhaghighi) [inline CHANGELOG.md:407]: What happened here?!

File: otherfile/RELEASE.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 	3. `SECURITY.md`
 	4. `otherfile/version_check.py`
 	5. `otherfile/meta.yaml`
-	6. `art/art_param.py`
+	6. `art/params.py`
 	7. `ArtList.ipynb`
 	8. `DecorList.ipynb`
 	9. `FontList.ipynb`
```
Comment (sepandhaghighi) [inline otherfile/RELEASE.md:16]: Update `help_func` function output in `tests\test.py`

File: otherfile/RELEASE.md
Comment (sepandhaghighi) [inline otherfile/RELEASE.md:1]: Update `last update` section

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 🔥 
Please review my comments and make sure to update `.github/CONTRIBUTING.md`

File: CHANGELOG.md (hunk 1 above)
Comment (sadrasabouri) [reply CHANGELOG.md]: I don't have any idea :confused: 
Will revert it.

File: otherfile/RELEASE.md
Comment (sadrasabouri) [reply otherfile/RELEASE.md:1]: fixed in [26db207](https://github.com/sepandhaghighi/art/pull/275/commits/26db20749129c4f82425c02d99a29ef53e191399)

**Review by sepandhaghighi — APPROVED** [review]
🔥

**Pull request merged**